### PR TITLE
improve(dataworker): Print out slow leaf execution args if ignoring

### PR DESCRIPTION
**Pull request by nicholaspai**
This way we can more easily manually execute the leaf to quiet alerts, or pass the `recipient` the execution args to execute themselves

### CODE REVIEW

File: src/dataworker/Dataworker.ts [hunk 1]
```diff
@@ -993,12 +993,25 @@ export class Dataworker {
...
+            relayData.amount,
+            relayData.originChainId,
+            relayData.realizedLpFeePct,
+            relayData.relayerFeePct,
+            relayData.depositId,
+            rootBundleId,
+            relayData.message,
+            payoutAdjustmentPct,
+            slowRelayTree.getHexProof({ relayData, payoutAdjustmentPct }),
+          ],
```
Comment (james-a-morris) [inline src/dataworker/Dataworker.ts:1014]: +1 for printing only the necessary items. 

Nit: would this make sense to include in the FormattingUtils?

**Review by james-a-morris — COMMENTED** [review]
Left a nit

File: src/dataworker/Dataworker.ts (hunk 1 above)
Comment (nicholaspai) [reply src/dataworker/Dataworker.ts:1014]: which part would be included in `FormattingUtils`?

Comment (james-a-morris) [reply src/dataworker/Dataworker.ts:1014]: My thought would be the code from 1002 to 1013 (the code that decomposes the object into an array)

**Pull request merged**